lib/trello/net.rb

```diff
@@ -16,23 +16,30 @@ def try_execute(request)
         begin
           if request
             result = execute_core request
-            Response.new(200, {}, result)
+            response_body = if [200, 201].include? result.status
+              result
+            else
+              result.body
+            end
+            Response.new(result.status, {}, response_body)
           end
-        rescue RestClient::Exception => e
-          raise if !e.respond_to?(:http_code) || e.http_code.nil?
-          Response.new(e.http_code, {}, e.http_body)
+        rescue Faraday::Error => e
+          raise if !e.respond_to?(:status) || e.status.nil?
+          Response.new(e.status, {}, e.body)
         end
       end
```

Comment:
I just realized I can simplify this with `faraday.response :raise_error`. I'll give that a try.